keycloak_oauth.gemspec

```diff
@@ -9,7 +9,7 @@ Gem::Specification.new do |spec|
   spec.summary       = %q{Implementing OAuth with Keycloak in Ruby}
   spec.homepage      = "https://rubygems.org/gems/keycloak_oauth"
   spec.license       = "MIT"
-  spec.required_ruby_version = Gem::Requirement.new(">= 2.3.0")
+  spec.required_ruby_version = Gem::Requirement.new(">= 2.6.0")
```

Comment:
This constitutes a breaking change, which normally would be reflected in a new major version for the gem.

Reply:
sure, will do!